### ISSUE TITLE
[Feature] Clickable tag

### ISSUE THREAD

**Issue by AnyoneMouse** [quillpad/quillpad#298]
Make tags clickable on notes. So that we can easily open tagged notes.

![Screenshot_20240419-232958.jpg](https://github.com/quillpad/quillpad/assets/85461823/f6d92031-d860-443f-b087-8adc6ca12d12)

**Comment by Gaalpos** [comment]
Yep, this need to be integrated , so tags are more useful and easier to use
